app/components/RegisterSingleTree/SingleTreeOverview.tsx

```diff
@@ -1039,6 +1118,22 @@ const SingleTreeOverview = () => {
               btnText={i18next.t('label.tree_review_next_btn')}
             />
           </View>
+        ) : showRemeasurementButton ? (
+          <View style={[styles.bottomBtnsContainer, { flexDirection: 'column' }]}>
+            <PrimaryButton
+              onPress={() => {
+                onPressRemeasure(route.params.item, `${route.params.sampleTreeIndex}`);
```

Comment:
I got an error `TypeError: undefined is not an object (evaluating 'route.params.item')` after having added and upload a single tree and then hit the "Remeasure" button on the detail screen of the new single tree:

![Simulator Screen Shot - iPhone 12 - 2023-01-21 at 12 29 53](https://user-images.githubusercontent.com/1532418/213865004-2a4a8ad3-76b6-42cf-888e-7f29854a0a8a.png)